state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -434,17 +482,28 @@ impl DataSummaryPoller {
             let storage_summary = match result {
                 Ok(storage_summary) => storage_summary,
                 Err(error) => {
-                    trace!(
-                        "Error encountered when polling peer ({:?})! Error: {:?}",
-                        peer,
-                        error
+                    error!(
```

Comment:
I also log the errors inside `send_request_to_peer()` when a response callback encounters an error, so this logging here may be not necessary anymore.